### ISSUE TITLE
Media Library: Add new button and thumbnail size slider are missing focus visual indicators

### ISSUE THREAD

**Issue by dpasque** [Automattic/wp-calypso#52222]
### Steps to reproduce the behavior

1. Go to the Media Library
2. Tab to the "Add new" button and the thumbnail size slider
3. Note the lack of focus indicator

This is also true for the images, but that is being tracked in issue #42412

#### What I expected to happen
There to be the same blue outline focus indicator as the other buttons

#### What actually happened
There was none

#### Browser / OS version
Chrome 90.0.4430.72
Windows 10 Home 19042.928

#### Screenshot / Video: If applicable, add screenshots to help explain your problem.
In this picture, the Add new doesn't have the focus state, but it shows what the focus state should match:
![focus_indicator](https://user-images.githubusercontent.com/26772342/115814007-e0c1b300-a3b9-11eb-8d7f-2ff56f1c1eea.png)
